README.md

```diff
@@ -66,9 +66,50 @@ contract MyContract is Ownable {
 }
 ```
 
+## Architecture
+The following provides visibility into how OpenZeppelin's contracts are organized:
+
+- **access** - Smart contracts that enable functionality that can be used for selective restrictions and basic authorization control functions. Includes address whitelisting and signature-based permissions management.
+- **crowdsale** - A collection of smart contracts used to manage token crowdsales that allow investors to purchase tokens with ETH. Includes a base contract which implements fundamental crowdsale functionality in its simplest form. The base contract can be extended in order to satisfy your crowdsale’s specific requirements.
+	- **distribution** - Includes extensions of the base crowdsale contract which can be used to customize the completion of a crowdsale.
+	- **emission** - Includes extensions of the base crowdsale contract which can be used to mint and manage how tokens are issued to purchasers.
+	- **price** - Includes extensions of the crowdsale contract that can be used to manage changes in token prices.
+	- **validation**  - Includes extensions of the crowdsale contract that can be used to enforce restraints and limit access to token purchases.
+- **examples** - A collection of simple smart contracts that demonstrate how to add new features to base contracts through multiple inheritance.
+- **introspection**  - An interface that can be used to make a contract comply with the ERC-165 standard as well as a contract that implements ERC-165 using a lookup table.
+- **lifecycle** - A collection of base contracts used to manage the existence and behavior of your contracts and their funds.
+- **math** - Libraries with safety checks used to prevent against overflows and underflows.
+- **mocks** - A collection of abstract contracts that can be used as the foundation for your own custom implementations. The mock contracts demonstrate how OpenZeppelin’s secure base contracts can be used with multiple inheritance.
+- **ownership** - A collection of smart contracts that can be used to manage contract and token ownership
+	- **rbac** - A library used to manage addresses assigned to different user roles and an example RBAC interface that demonstrates how to handle setters and getters for roles and addresses.
+
+- **payment** - A collection of smart contracts that can be used to manage payments through escrow arrangements, withdrawals, and claims. Includes support for both single payees and multiple payees.
+- **proposals** - A collection of smart contracts that reflect community Ethereum Improvement Proposals (EIPs). The smart contracts here are under consideration for inclusion in OpenZeppelin. They are not recommended for use until the community has thoroughly evaluated and approved them.
```

Comment:
let's add a link to https://github.com/OpenZeppelin/openzeppelin-solidity/wiki/ERC-Process so people can get the full details.

Maybe instead of "not recommended for use..." we can say something like "these contracts are under development and standardization, so they're not recommended for production contracts, but are useful for experimentation with pending EIP standards."